docs/photoniq/event-delivery/clients/connection.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 10
+title: Connection
+---
+
+It maintains a WebSocket/SSE connection:
+
+## Supported methods
+
+### connect
+
```

Comment:
What is the intent of this section?

You define methods.  But I don't know what to do with them.  For example, getConfig() returns a Config.
Is Config an opaque type?  Or is there information inside it that I need?

I don't understand, as a user of the Client SDK how can I use this information that is being provided here?

Reply:
OK, I found that config is defined on the main "Client SDKs" page.

Still, when I am looking at SDKs, I want to look at all the parameters.  This does not define the APIs well.

We are going to need to get help from the doc person to give their thoughts.

Reply:
I'm still familiarizing myself with the docs, but I agree. Some users may get frustrated. Let's make it easier for them to find everything they need in one place and link to what needs any extra definition. 

How about after defining the methods, we add any required parameters and show what calling that method returns? This helps let them know if they're on the right track.

Reply:
@joubert-macrometa - it is not clear how to improve. Do you mean to provide link to `Config` Schema table for `getConfig()` method? Or even put all methods into a table?

Reply:
@dlazarenko-macrometa I am also not 100% sure.  Just what we have now is not great.

What @maryann-macrometa is saying is probably close to what we need.  Mimic what swagger is doing.  Knowing what the inputs are and what the outputs are in one place will make, me as a developer, much happier :-) 

@maryann-macrometa as you come up to speed on the docs, keep an eye if we have done something like this before.  Documenting the SDK.  Maybe we leverage that for now until you can help us come up with a more coherent strategy for this.

Reply:
@joubert-macrometa I'll do that.

Reply:
@joubert-macrometa, @maryann-macrometa - I added link to `Config`